src/components/MultiSelect/constants.ts

```diff
@@ -0,0 +1,22 @@
+export const OPTIONS = [
```

Comment:
Maybe it will be simpler?
```js
const OPTIONS = ["Crime", "Documentary", "Horror", "Comedy"];

export OPTIONS.map((item) => ({
  value: item,
  label: item,
  isChecked: item
}));
```

Reply:
Thanks, this is fixed